resources/views/partials/content-page-saved-searches.blade.php

```diff
@@ -0,0 +1,6 @@
+<div class="align-right" hidden><button id="remove-all" class="button button--borderless button--destructive">@svg('delete', 'icon--delete', ['focusable' => 'false', 'aria-hidden' => 'true']) {{ __('Remove all', 'coop-library') }}<span class="screen-reader-text"> {{ __('saved-searches', 'coop-library') }}</span></button></div>
```

Comment:
The screen reader text isn't quite right. Currently says: "Remove allsaved-searches".
* Missing space between "all" and "saved"
* remove hyphen from "saved-searches".

Reply:
@jhung there is a space inside the `<span>` but it appears not to be read properly by NVDA. I've tried to add one outside in 2314ee7 in case that works better (I'm using a zero-width space to avoid adding a weird extra margin at the end of the button).